internals/cli/cmd_restart.go

```diff
@@ -34,11 +28,13 @@ type cmdRestart struct {
 
 func init() {
 	AddCommand(&CmdInfo{
-		Name:        "restart",
-		Summary:     shortRestartHelp,
-		Description: longRestartHelp,
-		Builder:     func() flags.Commander { return &cmdRestart{} },
-		OptionsHelp: waitOptionsHelp,
+		Name:    "restart",
+		Summary: "Restart a service",
+		Description: `
+The restart command restarts the named service(s) in the correct order.
+`,
+		ArgsHelp: waitArgsHelp,
```

Comment:
Considering there is re-use of these argsHelp maps, I think it would make more sense for them to be sets of a specific type describing one option, rather than maps. Doing it like this would also allow you to put more complex information into it: options with (potentially multiple and/or optional) arguments, shorthand variations, repeatability etc. and it would also clarify the way that the options are shared in-between commands.